auth-server/src/model/PasswordResetToken.ts

```diff
@@ -3,45 +3,38 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Model } from 'objection';
+
+import { JsonSchema } from '../types';
 import connection from './connection';
 
 Model.knex(connection);
 
 export default class PasswordResetToken extends Model {
+	expires!: string
 	readonly id!: number
 	token!: string
 	user_id!: number
-	valid: boolean
-	expires: string
+	valid!: boolean
 
-	static get tableName () {
+	static get tableName (): string {
 		return 'password_reset_token';
 	}
 
-	static get idColumn () {
+	static get idColumn (): string {
```

Comment:
hmm this should be number.
```suggestion
	static get idColumn (): number {
```

Reply:
its retuning column name return 'id'; which is a string